processor/nameprocessor/README.md

```diff
@@ -0,0 +1,32 @@
+# Name processor
```

Comment:
High level comment, there's already a processor for span names, I think adding another one will cause confusion

https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor/spanprocessor

Name sanitization seems like a nice feature to add to that processor so that would be better.

Reply:
Thanks. Will check that out.